sci-libs/matio/matio-1.5.19.ebuild

```diff
@@ -8,7 +8,7 @@ HOMEPAGE="https://sourceforge.net/projects/matio/"
 SRC_URI="mirror://sourceforge/${PN}/${P}.tar.gz"
 
 LICENSE="BSD-2"
-SLOT="0/9" # subslot = soname version
+SLOT="0/11" # subslot = soname version
```

Comment:
`find "${ED}" -name "*.la" -delete || die` in `src_install`

Reply:
Done